public/demo.js

```diff
@@ -348,6 +352,11 @@ $(document).ready(function() {
         });
     }
 
+    function getSrcLang() {
+        var x = $('#dropdownMenuInput').text().split(" ");
+        return x[0];
```

Comment:
Why split and return the first word? I think this will break the check for "language" on line 176 because the first word is "detect".

I would suggest that this method normalize the text if desired (trim and/or lowercase), and if it is set to "detect language" then return null or "" or something along those lines. Then line 176 should check for a "truthy" value rather than looking for "language". (I know that part was already there, but if we're going to change it, we might as well do it right. ;)